lambdas/build-proven-user-identity-details/src/main/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/BuildProvenUserIdentityDetailsHandler.java

```diff
@@ -153,10 +154,12 @@ private NameAndDateOfBirth getProvenIdentityNameAndDateOfBirth(
             throws ParseException, JsonProcessingException, ProvenUserIdentityDetailsException,
                     NoVcStatusForIssuerException {
         for (VcStoreItem item : credentialIssuerItems) {
+            CredentialIssuerConfig credentialIssuerConfig =
+                    configService.getCredentialIssuerActiveConnectionConfig(
+                            item.getCredentialIssuer());
             if (EVIDENCE_CRI_TYPES.contains(item.getCredentialIssuer())
                     && userIdentityService.isVcSuccessful(
-                            currentVcStatuses,
-                            configService.getComponentId(item.getCredentialIssuer()))) {
+                            currentVcStatuses, credentialIssuerConfig.getComponentId())) {
```

Comment:
I'd be inclined to stick with just calling `configService.getComponentId(item.getCredentialIssuer())`. It does the same thing under the hood (gets the active connection to get the config) but is a bit less verbose than fetching the config directly. Also, there might be a future where we don't want to use the active connection but would rather use whatever connection the user initiated a journey with a CRI with.

Reply:
Done.